app/static/js/events/scheduler.js

```diff
@@ -342,9 +342,10 @@ function addSessionToUnscheduled(sessionRef, isFiltering, shouldBroadcast) {
     }
 
     try {
-        sessionRefObject.$sessionElement.popover('destroy');
-    } catch(ignored) { }
-
+        setTimeout( function() {
+            $('.session.unscheduled').popover('hide');}, 100);
```

Comment:
The delay is added because the issue was caused in the first place  due to asynchronous code.
And the popover('destroy'), did not delete the content inside the popover.